module-apps/application-call/ApplicationCall.cpp

```diff
@@ -281,10 +280,11 @@ namespace app
         }
     }
 
-    sys::MessagePointer ApplicationCall::handleAudioMessageEvent(AudioEventRequest *message)
+    sys::MessagePointer ApplicationCall::handleAudioMessageEvent()
```

Comment:
It would be much better if we have a specific AudioEvent that correctly notifies us about audio profile changes.